### PR TITLE
Add basic functionality to Discord bot

### LINKED ISSUE #6: Add basic functionality to Discord bot
Add several basic functions to the Discord bot, including the ability to use slash commands.

### PR DESCRIPTION
**Pull request by AlexisMLopez**
Add several basic functions to the Discord bot, including the ability to use slash commands.

### CODE REVIEW

File: discord-script.py [hunk 1]
```diff
@@ -1,19 +1,67 @@
...
-bot = commands.Bot(command_prefix="!", intents=discord.Intents.all())
+class DiscordBot(discord.Client):
+    def __init__(self):
+        super().__init__(intents=discord.Intents.default())
+        self.synced = False
+
+    # bot syncs slash commands on startup
+    async def on_ready(self):
+        await tree.sync(guild=discord.Object(
+            id=1030163730538954853))  # without this the slash commands will not instantly update (encrypt later)
```
Comment (RonnyFrayRegato) [inline discord-script.py]: The id parameter's value should be converted into a variable since it's used several times throughout the file.

Comment (RonnyFrayRegato) [inline discord-script.py]: Does the Discord server ID need to be encrypted for security?

Comment (AlexisMLopez) [reply discord-script.py]: The server ID variable has been added.

Comment (AlexisMLopez) [reply discord-script.py]: It does not because server IDs are public to anyone that owns a Discord account.

**Review by zachary-mixson — APPROVED** [review]
looks good!

**Review by RonnyFrayRegato — APPROVED** [review]

**Pull request merged**